lib/run-status.js

```diff
@@ -126,10 +132,11 @@ class RunStatus extends EventEmitter {
 		this.tests = flatten(this.tests);
 		this.passCount = sum(this.stats, 'passCount');
 		this.knownFailureCount = sum(this.stats, 'knownFailureCount');
+		this.pendingCount = sum(this.stats, 'pendingCount');
 		this.skipCount = sum(this.stats, 'skipCount');
 		this.todoCount = sum(this.stats, 'todoCount');
 		this.failCount = sum(this.stats, 'failCount');
-		this.remainingCount = this.testCount - this.passCount - this.failCount - this.skipCount - this.todoCount - this.knownFailureCount;
+		this.remainingCount = this.testCount - this.passCount - this.failCount - this.skipCount - this.todoCount - this.knownFailureCount - this.pendingCount;
```

Comment:
At first blush you'd think this is now always `0`, but that wouldn't be the case with `--fail-fast` not emitting the pending tests. Perhaps a comment would help clarify that possibility.

Reply:
See follow-up [above][https://github.com/avajs/ava/pull/1596#discussion_r154272744].